ldapadmin/src/main/webapp/WEB-INF/spring/webmvc-config.xml

```diff
@@ -123,17 +123,13 @@
     <property name="driverClassName" value="org.postgresql.Driver"/>
   </bean>
 
-    <bean id="dataSource" class="org.apache.commons.dbcp.BasicDataSource" depends-on="waitForDb">
-        <property name="url" value="${psql.url}"/>
-        <property name="username" value="${psql.user}"/>
+    <bean id="dataSource" class="com.mchange.v2.c3p0.ComboPooledDataSource" depends-on="waitForDb">
+        <property name="jdbcUrl" value="${psql.url}"/>
+        <property name="user" value="${psql.user}"/>
         <property name="password" value="${psql.pass}"/>
-        <property name="driverClassName" value="org.postgresql.Driver"/>
-        <property name="testOnBorrow" value="true"/>
-        <property name="validationQuery" value="select 1 as dbcp_connection_test"/> 
-        <property name="poolPreparedStatements" value="true"/>
-        <property name="maxOpenPreparedStatements" value="-1"/>
-        <property name="defaultReadOnly" value="false"/>
-        <property name="defaultAutoCommit" value="true"/>
+        <property name="driverClass" value="org.postgresql.Driver"/>
+        <property name="maxPoolSize" value="5"/>
```

Comment:
Might be too short... let's make it 10 by default.

Reply:
It's LDAPAdmin ;-)